src/display/api.js

```diff
@@ -375,6 +378,9 @@ function getDocument(src) {
         isValidFetchUrl(params.cMapUrl, document.baseURI) &&
         isValidFetchUrl(params.standardFontDataUrl, document.baseURI));
   }
+  if (typeof params.takeOwnershipOfData !== "boolean") {
+    params.takeOwnershipOfData = false;
```

Comment:
nit: Isn't enough to write: `params.takeOwnershipOfData = params.takeOwnershipOfData  === true;` ?